docs: add thumbnail generator canvas example (#1102)
fusedio/fused-docs: docs/examples/thumbnail-generator.mdx

---
id: thumbnail-generator
title: Building a Repeatable Thumbnail Generator with Claude Code
sidebar_label: Thumbnail generator
sidebar_position: 16
---

> Every time we host a [LinkedIn Live webinar](https://www.linkedin.com/feed/update/urn:li:activity:7462432559440977920/), we need a thumbnail — speaker photos, title, brand colors, consistent layout. The first one took an afternoon of manual Figma work. By the third, that felt unreasonable. This example shows how we replaced that workflow with a Fused Canvas that anyone on the team can open, fill in, and download in under a minute.

Author: ```suggestion
Every time we host a [LinkedIn Live webinar](https://www.linkedin.com/feed/update/urn:li:activity:7462432559440977920/), we need a thumbnail — speaker photos, title, brand colors, consistent layout. This example shows how we replaced a repetitive workflow with a Fused Canvas that anyone on the team can open, fill in, and download in under a minute.
```
